### PR TITLE
moved date to bottom

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: unicorecmsgem/templates/base.pt [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <meta name="description" content=""/>
     <meta name="keywords" content="moms,maternal,baby,pregnancy,pregnant,expectant,mama,mother,pain,birth,help"/>
     <link href="${request.static_url('cms:static/css/style.css')}?v=20141113" rel="stylesheet" type="text/css" />
+    <link href="${request.static_url('cms:static/css/<inline style sheet #2>')}?v=20141113" rel="stylesheet" type="text/css" />
```
Comment (miltontony) [inline unicorecmsgem/templates/base.pt]: what's this?

File: unicorecmsgem/templates/base.pt [hunk 2]
```diff
@@ -25,7 +26,7 @@
           </span>
           </div>
         </div>
-        <div class="logo"><a href="/" class="logo"><img alt='Girl Effect Mobile' src="/static/img/logo.png" /></a></div>
+        <div class="logo"><a href="/" class="logo"><img alt='Girl Effect Mobile' src="/static/img/logo_x2.png" /></a></div>
```
Comment (miltontony) [inline unicorecmsgem/templates/base.pt:28]: You'll need to add `logo_x2.png` with `git add /path/to/logo_x2.png`

Comment (codiebeulaine) [inline unicorecmsgem/templates/base.pt:28]: cool, will do

File: unicorecmsgem/templates/content.pt [hunk 3]
```diff
@@ -1,31 +1,22 @@
...
-        <div class="detail">
-            <div tal:attributes="{'class':'head'} if page.image_host and page.image else {'class': 'head no-pic'}">
-                <condition tal:omit-tag="" tal:condition="page.image_host and page.image">
-                    <div class='img'>
-                    <a href="/content/detail/${page.uuid}/">
-                        <img alt='${page.title}' src="${view.get_image_url(page.image_host, page.image, 45, 45)}"/>
-                    </a>
+                <div tal:attributes="{'class':'head'} if page.image_host and page.image else {'class': 'head no-pic'}">
+                    <div tal:define="localisation view.get_localisation()" tal:omit-tag="">
+                         <condition tal:omit-tag="" tal:condition="localisation and localisation.image_host and localisation.image">
```
Comment (miltontony) [inline unicorecmsgem/templates/content.pt]: this needs to be the page image and not the localisation image. So it needs to be `page.image_host and page.image`

Comment (codiebeulaine) [inline unicorecmsgem/templates/content.pt]: done

File: unicorecmsgem/templates/content.pt [hunk 4]
```diff
@@ -1,31 +1,22 @@
...
 
-        <div class="detail">
-            <div tal:attributes="{'class':'head'} if page.image_host and page.image else {'class': 'head no-pic'}">
-                <condition tal:omit-tag="" tal:condition="page.image_host and page.image">
-                    <div class='img'>
-                    <a href="/content/detail/${page.uuid}/">
-                        <img alt='${page.title}' src="${view.get_image_url(page.image_host, page.image, 45, 45)}"/>
-                    </a>
+                <div tal:attributes="{'class':'head'} if page.image_host and page.image else {'class': 'head no-pic'}">
+                    <div tal:define="localisation view.get_localisation()" tal:omit-tag="">
```
Comment (miltontony) [inline unicorecmsgem/templates/content.pt]: You don't need this anymore

File: unicorecmsgem/templates/content.pt [hunk 5]
```diff
@@ -1,31 +1,22 @@
...
+                         <condition tal:omit-tag="" tal:condition="page.image_host and page.image">
+                            <img alt="content image" src="${view.get_image_url(page.image_host, page.image, 320)}"/>
```
Comment (miltontony) [inline unicorecmsgem/templates/content.pt]: Please use the title for the `alt` text `alt='${page.title}'`

File: unicorecmsgem/templates/base.pt (hunk 1 above)
Comment (codiebeulaine) [inline unicorecmsgem/templates/base.pt]: Oh, sorry that was me before I vaguely understood what I was doing

File: unicorecmsgem/templates/home.pt [hunk 6]
```diff
@@ -31,6 +31,9 @@
                 Latest
             </div>
             <div class="listing no-pic" tal:repeat="page view.get_featured_pages()">
```
Comment (miltontony) [inline unicorecmsgem/templates/home.pt]: @codiebeulaine you'll have to do some `tal:attributes` for the `class` here.

File: unicorecmsgem/templates/home.pt [hunk 7]
```diff
@@ -42,6 +45,9 @@
                 <a href="/content/list/${cat.uuid}/">${cat.title}</a>
             </div>
             <div class="listing no-pic" tal:repeat="page view.get_featured_category_pages(cat.uuid)">
```
Comment (miltontony) [inline unicorecmsgem/templates/home.pt]: same here.

**Comment by miltontony** [comment]
:+1:  once the `no-pic` has been handled

File: unicorecmsgem/templates/content.pt [hunk 8]
```diff
@@ -1,31 +1,18 @@
...
-                    <a href="/content/detail/${page.uuid}/">
-                        <img alt='${page.title}' src="${view.get_image_url(page.image_host, page.image, 45, 45)}"/>
-                    </a>
-                    </div>
-                </condition>
-                <div class="meta">${view.format_date(page.created_at)}</div>
-                <div class="h2">${page.title}</div>
-                <div class="meta">${page.description}</div>
-            </div>
+                <div tal:attributes="{'class':'head'} if page.image_host and page.image else {'class': 'head no-pic'}">
```
Comment (miltontony) [inline unicorecmsgem/templates/content.pt:7]: @codiebeulaine you need something like this

Comment (codiebeulaine) [inline unicorecmsgem/templates/content.pt:7]: @miltontony  not quite sure if i implemented it correctly?

**Comment by miltontony** [comment]
:+1:

**Pull request closed without merging**